dev-support/cicd/HowToRelease.md

```diff
@@ -95,7 +98,7 @@ docker tag local/mini-submarine:0.3.0 apache/submarine:mini-0.3.0-RC0
 ```
 In the container, we can verify that the submarine jar version is the expected 0.3.0. Then we can upload this image with a "RC" tag for a vote.
 ```
-docker push apache/mini-submarine:0.3.0:RC0
+docker push apache/submarine:mini-0.3.0-RC0
```

Comment:
Does it require an INFRA ticket to grant you permission to the ASF DockerHub? like this one? https://issues.apache.org/jira/browse/INFRA-20364